astropy/coordinates/tests/test_arrays.py

```diff
@@ -67,6 +67,17 @@ def test_angle_from_pyarrow():
     npt.assert_array_equal(angle.value, input_data)
 
 
+def test_angle_from_pandas():
```

Comment:
I think this is better marked with 
```
@pytest.mark.skipif(not HAS_PANDAS, reason="Requires pandas")
```
Of course, then it also needs
```
from astropy.utils.compat.optional_deps import HAS_PANDAS
```
on top if it isn't there already.

Reply:
I think we should define a project-level convention for this, @eerovaher asked for the exact inverse of this change on another recent PR of mine. I don't have a strong preference myself so I'm happy to go with whoever reviews first but overall this seems inefficient.

Reply:
(On my phone right now but I'll open an issue for this later today. In the meantime I'd suggest we accept both style)

Reply:
There is a project-level guide somewhere that states that one should follow the sub-module's standard practice. `coordinates` has exactly one `importorskip` (added very recently, I think; maybe the one you mentioned) and lots of `HAS_`, so I think we should just go for that.

FWIW, my main logic is that the body of the tests should not distract understanding; for the same reason, I rather dislike pytest fixtures, which means that one has to look up the fixture, likely somewhere quite distant, to understand what actually is happening.

Reply:
All the above said, as long as one actually uses the module, which is the case here, it does make things shorter, since with `HAS_PYARROW` one still would have to have an import statement. So, let's just go with it here at least.